CHANGELOG.md

```diff
@@ -21,7 +21,21 @@ PLEASE REMEMBER TO CHANGE THE '..master' WITH AN ACTUAL TAG in GITHUB LINK.
 
     There is no longer a separate jaxlib release for CUDA 11.2 (or higher); use
     the CUDA 11.1 wheel for those versions (cuda111).
-  * Added support for static keyword arguments to the C++ `jit` implementation.
+  * Jaxlib now bundles `libdevice.10.bc` in CUDA wheels. There should be no need
+    to point JAX to a CUDA installation to find this file.
+  * Added support for static keyword arguments to the C++ {func}`jit`
+    implementation.
```

Comment:
```suggestion
  * Added automatic support for static keyword arguments to the {func}`jit`
    implementation.
```

(the Python feature doesn't launch until the jaxlib update!)

Reply:
Done.